src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/RepoInfoWithStatus.java

```diff
@@ -42,6 +40,9 @@
   /** If this value is set then indicates that this repo had an error when being created. */
   private String errorMessage;
 
+  /** If this value is set then indicates that this repo has a custom problem statement. */
+  private String problemStatement;
```

Comment:
How about always setting this even if its not a custom problem statement?

Reply:
We could but it's a bit fiddly as we'll have to read task every time to load the RepoInfo? Or I could always store a copy of the problemStatement?

Reply:
Yeah. Let's always store it - then we know that all repos contain a problem statement and can treat them all the same.

Reply:
Done.